### ISSUE TITLE
Error when execute 'insert syntax' using DB type sequence with 'processors=1' in server.xml

### ISSUE THREAD

**Issue by zhunina** [actiontech/dble#348]
```
mysql> insert into t5 values('aa');
ERROR 1064 (HY000): can't fetch sequence in db,sequence :`testdb`.`t5` detail:null
```
```
<property name="processors">1</property>
```

**Comment by zhunina** [comment]
```
2017-11-30 11:17:29.611  WARN [$_NIOREACTOR-0-RW] (com.actiontech.dble.server.ServerConnection.executeException(ServerConnection.java:367)) - ServerConnection [id=1, schema=testdb, host=127.0.0.1, user=test,txIsolation=2, autocommit=true, schema=testdb]insert into t5 values('aa') err:java.lang.RuntimeException: can't fetch sequence in db,sequence :`testdb`.`t5` detail:null
java.lang.RuntimeException: can't fetch sequence in db,sequence :`testdb`.`t5` detail:null
        at com.actiontech.dble.route.sequence.handler.IncrSequenceMySQLHandler.getSeqValueFromDB(IncrSequenceMySQLHandler.java:105) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.sequence.handler.IncrSequenceMySQLHandler.nextId(IncrSequenceMySQLHandler.java:73) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.parser.druid.impl.DruidInsertParser.appendValues(DruidInsertParser.java:460) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.parser.druid.impl.DruidInsertParser.convertInsertSQL(DruidInsertParser.java:376) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.parser.druid.impl.DruidInsertParser.visitorParse(DruidInsertParser.java:86) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.parser.druid.impl.DefaultDruidParser.parser(DefaultDruidParser.java:47) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.util.RouterUtil.routeFromParser(RouterUtil.java:119) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.impl.DefaultRouteStrategy.routeNormalSqlWithAST(DefaultRouteStrategy.java:59) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.impl.AbstractRouteStrategy.route(AbstractRouteStrategy.java:60) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.route.RouteService.route(RouteService.java:108) ~[dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.server.ServerConnection.routeEndExecuteSQL(ServerConnection.java:306) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.server.ServerConnection.execute(ServerConnection.java:250) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.server.ServerQueryHandler.query(ServerQueryHandler.java:137) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.FrontendConnection.query(FrontendConnection.java:314) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.FrontendConnection.query(FrontendConnection.java:332) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.handler.FrontendCommandHandler.handleData(FrontendCommandHandler.java:51) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.handler.FrontendCommandHandler.handle(FrontendCommandHandler.java:40) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.FrontendConnection.rawHandle(FrontendConnection.java:469) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.FrontendConnection.handle(FrontendConnection.java:451) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.AbstractConnection.onReadData(AbstractConnection.java:297) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.NIOSocketWR.asyncRead(NIOSocketWR.java:175) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.AbstractConnection.asyncRead(AbstractConnection.java:256) [dble-2.17.11.0-dev.jar:?]
        at com.actiontech.dble.net.NIOReactor$RW.run(NIOReactor.java:73) [dble-2.17.11.0-dev.jar:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_151]
```

**Comment by yanhuqing666** [comment]
fixed at c3f22ff6ebf83808f6524dc7e8a91e2fe9ab14da

**Comment by zhunina** [comment]
fixed
